### PR TITLE
Docs 2861: Unmatched Bank Transfer payments

### PR DESCRIPTION
**Pull request by FrancescaNCook**
Do the changes meet MultiSafepay Docs' Definition of Done?
- Style guide, tone of voice, and naming conventions
- Tested
- Reviewed on mobile, tablet and desktop screens
- Notified stakeholders
- Pass GitHub Actions

### CODE REVIEW

**Comment by FrancescaNCook** [comment]
With Lisa for review.

File: content/payments/methods/banks/bank-transfer/user-guide/unmatched-payments-dutch.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+meta_title: "Ongematchte bankoverschrijvingen - MultiSafepay Docs"
+read_more: "."
+weight: 1
+url: '/bank-transfer/ongematchte-bankoverschrijvingen/'
+tags: "hidden"
+---
+
+Lees deze pagina ook in het [Engels](/bank-transfer/unmatched-payments/) of [Duits](/bank-transfer/unzugeordneten-banküberweisungen/).
+
+Wanneer MultiSafepay een bankoverschrijving ontvangt van een klant, matchen we die automatisch aan de bijbehorende transactie in ons systeem. We doen dit op basis van de betalingsgegevens.
```
Comment (Kris-MultiSafepay) [inline content/payments/methods/banks/bank-transfer/user-guide/unmatched-payments-dutch.md:12]:
```suggestion
Wanneer MultiSafepay een bankoverschrijving ontvangt van een klant, matchen we die automatisch aan de bijbehorende transactie in ons systeem. We doen dit op basis van:

 - Het bedrag en betalingskenmerk, **of**
 - Het bedrag en bankrekeningnummer van de klant
```

File: content/payments/methods/banks/bank-transfer/user-guide/unmatched-payments-dutch.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+- Geklikt op **Bevestig** op de MultiSafepay betaalpagina (redirect orders). 
+
+Beide gevallen leiden ertoe dat er geen transactie bestaat in ons systeem waarmee de overschrijving gematcht kan worden.
+
+Om een transactie aan te maken kunt u gebruik maken van onze betaallinks ([payment links](/payment-links/)):
+
+### Er bestaat al een betaallink
+
+1. Klik op de link om de betaalpagina te openen. 
+2. Als de klant het veld **Uw bankrekeningnummer** niet heeft ingevuld, vul dan het bankrekeningnummer van de klant in.
```
Comment (Kris-MultiSafepay) [inline content/payments/methods/banks/bank-transfer/user-guide/unmatched-payments-dutch.md:41]:
```suggestion
2. Als de klant het veld **Bankrekeningnummer** niet heeft ingevuld, vul dan het bankrekeningnummer van de klant in (indien bij u bekend).
```

File: content/payments/methods/banks/bank-transfer/user-guide/unmatched-payments-dutch.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+3. Klik op **Bevestig** om de transactie aan te maken in ons systeem.
+
+### Er bestaat nog geen betaallink
+
+1. [Genereer handmatig een betaallink](/payment-links/generating-links/). 
+2. Voeg de naam van de klant en het ordernummer toe aan de beschrijving (voor uw eigen administratie). 
+3. Klik op **Bankoverschrijving**.
```
Comment (Kris-MultiSafepay) [inline content/payments/methods/banks/bank-transfer/user-guide/unmatched-payments-dutch.md:48]:
```suggestion
3. Klik op **Bankoverschrijving**.
4. Voeg het bankrekeningnummer van de klant toe (indien bij u bekend), zodat we de betaling gemakkelijker kunnen matchen.
```

File: content/payments/methods/banks/bank-transfer/user-guide/unmatched-payments-dutch.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+4. Klik op **Bevestig** om de transactie aan te maken in ons systeem.
```
Comment (Kris-MultiSafepay) [inline content/payments/methods/banks/bank-transfer/user-guide/unmatched-payments-dutch.md:49]:
```suggestion
5. Klik op **Bevestig** om de transactie aan te maken in ons systeem.
```

**Review by Kris-MultiSafepay — COMMENTED** [review]
Adjustments to Dutch text

File: content/payments/methods/banks/bank-transfer/payment-flow.md [hunk 5]
```diff
@@ -23,37 +23,58 @@ sequenceDiagram
...
 
-## Payment statuses
+To prevent us from emailing the customer, in your `POST /orders` request, set the `disable_send_email` parameter to `true`. See API reference – [Bank Transfer](/api/#bank-transfer).
+
+## 2. Customer transfers funds
+
+The customer must only pay for **one** order per transfer. When transfering the funds, they must correctly input:  
+    
+- The amount
+- Their bank account number
```
[the remark this answers is not in the document]
Comment (FrancescaNCook) [reply content/payments/methods/banks/bank-transfer/payment-flow.md:58]: Their own bank account number in the redirect page.

File: content/payments/methods/banks/bank-transfer/_index.md [hunk 6]
```diff
@@ -1,16 +1,18 @@
...
-    - /payment-methods/banks/bank-transfer
-    - /payments/methods/banks/bank-transfer/
+---
+title: 'Bank Transfer'
+weight: 60
+meta_title: "Payment methods - Bank Transfer - MultiSafepay Docs"
+linktomarketing: "https://www.multisafepay.com/solutions/payment-methods/bank-transfer"
+childlist: '.'
+faq: '.'
+intro_description: "Bank Transfer, also known as SEPA Credit Transfer, is a secure, trusted, international banking method. Customers can make any type of online payment in euros within the SEPA area. You can link Bank Transfer directly to your website. When a customer selects Bank Transfer as payment method, they receive MultiSafepay bank account details by email.
```
Comment (FrancescaNCook) [inline content/payments/methods/banks/bank-transfer/_index.md:8]:
```suggestion
intro_description: "Bank Transfer, also known as SEPA Credit Transfer, is a secure, trusted, international banking method. Customers can make any type of online payment in euros within the SEPA area. 
```

**Pull request merged**